_pages/en/it-picture-long-term.md

```diff
@@ -99,11 +112,18 @@ EA teams provide a loosely coupled architecture (reusable infrastructure artifac
 EA is held accountable to:
 
 - Provide an artifact repository which, when used by other teams, puts them into compliance
-- Other teams may freely opt out of their services, and they will be measured against the uptake rate from their clients
+- Produce reference architectures for solution teams to use and align their designs to
```

Comment:
We need to automate EA, not have them just produce documentation as they do now ... no?

Reply:
I disagreed, partly. The trend is to light reference architectures. I will look for references to support.
It's still DevOps. But devOps is using reference arch. Look at AWS and how many ref arch they produce to help developers use their services.

Reply:
See commit #02a1519 with wording changes to add DevOps and common vision